cds_dojson/marc21/fields/videos/video.py

```diff
@@ -140,9 +141,21 @@ def location(self, key, value):
 
 
 @model.over('internal_note', '^595__')
+@ignore_value
 def internal_note(self, key, value):
     """Internal note."""
-    return ", ".join(filter(None, [value.get('a'), value.get('s')]))
+    _internal_categories = defaultdict(list)
+    _internal_categories.update(self.get('internal_categories', {}))
+    _internal_notes = self.get('internal_note', '').splitlines()
+    for v in force_list(value):
+        if v.get('a') in ('CERN50', 'CERN EDS', 'Video-SR-F', 'Pilote PICTURAE', 'Press'):
```

Comment:
does it make sense maybe to pout this in config.py?

Reply:
I don't know, we are just going to run it now and forget about it ...